dll/cpl/sysdm/environment.c

```diff
@@ -1842,6 +1871,12 @@ EnvironmentDlgProc(HWND hwndDlg,
             }
             SetWindowLongPtr(hwndDlg, DWLP_USER, (LONG_PTR)DlgData);
 
+            INT len = GetDlgItemTextW(hwndDlg, IDC_USER_VARIABLE_GROUP, szFormat, Length);
+            Length -= len;
```

Comment:
Since ``len`` isn't used anywhere else, this could be simplified into:
```suggestion
            Length -= GetDlgItemTextW(hwndDlg, IDC_USER_VARIABLE_GROUP, szFormat, Length);
```